src/Private/Responses.ps1

```diff
@@ -48,4 +82,429 @@ function Show-PodeErrorPage {
+            }
+            $htmlContent.Append("</td> <td class='dateTime'>")
+            $htmlContent.Append($item.CreationTime.ToString('yyyy-MM-dd HH:mm:ss'))
+            $htmlContent.Append("</td> <td class='dateTime'>")
+            $htmlContent.Append($item.LastWriteTime.ToString('yyyy-MM-dd HH:mm:ss'))
+            $htmlContent.Append( "</td> <td class='size'></td> <td class='icon'><i class='bi bi-folder2-open'></td> <td class='name'><a href='")
+            $htmlContent.Append($ParentLink)
+            $htmlContent.AppendLine("'>..</a></td> </tr>")
+        }
+        # Retrieve the child items of the specified directory
+        $child = Get-ChildItem -Path $Path -Force
+        foreach ($item in $child) {
+            $link = "$baseLink/$([uri]::EscapeDataString($item.Name))"
+            if ($item.PSIsContainer) {
+                $size = ''
+                $icon = 'bi bi-folder2'
+            }
+            else {
+                $size = '{0:N2}KB' -f ($item.Length / 1KB)
+                $icon = 'bi bi-file'
+            }
+
+            # Format each item as an HTML row
+            if ($windowsMode) {
+                $htmlContent.Append("<tr> <td class='mode'>")
+                $htmlContent.Append($item.Mode)
+            }
+            else {
+                $htmlContent.Append("<tr> <td class='unixMode'>")
+                $htmlContent.Append($item.UnixMode)
+                $htmlContent.Append("</td> <td class='user'>")
+                $htmlContent.Append($item.User)
+                $htmlContent.Append("</td> <td class='group'>")
+                $htmlContent.Append($item.Group)
+            }
+            $htmlContent.Append("</td> <td class='dateTime'>")
+            $htmlContent.Append($item.CreationTime.ToString('yyyy-MM-dd HH:mm:ss'))
+            $htmlContent.Append("</td> <td class='dateTime'>")
+            $htmlContent.Append($item.LastWriteTime.ToString('yyyy-MM-dd HH:mm:ss'))
+            $htmlContent.Append("</td> <td class='size'>")
+            $htmlContent.Append( $size)
+            $htmlContent.Append( "</td> <td class='icon'><i class='")
+            $htmlContent.Append( $icon)
+            $htmlContent.Append( "'></i></td> <td class='name'><a href='")
+            $htmlContent.Append( $link)
+            $htmlContent.Append( "'>")
+            $htmlContent.Append($item.Name )
+            $htmlContent.AppendLine('</a></td> </tr>' )
+        }
+
+        $Data = @{
+            RootPath    = $RootPath
+            Path        = $leaf.Replace('\', '/')
+            WindowsMode = $windowsMode.ToString().ToLower()
+            FileContent = $htmlContent.ToString()   # Convert the StringBuilder content to a string
+        }
+
+        $podeRoot = Get-PodeModuleMiscPath
+        # Write the response
+        Write-PodeFileResponseInternal -Path ([System.IO.Path]::Combine($podeRoot, 'default-file-browsing.html.pode')) -Data $Data
+    }
+    catch {
+        write-podehost $_
```

Comment:
I would just let the error get bubbled up the chain, and remove the try/catch. This way the error is caught and appropriately logged and reported back to the user as a 500

Reply:
It was for debug and I forgot to remove it